File: rb/utility/make_context.rb
# IpGeolocationApi3 SDK utility: make_context
require_relative '../core/context'
module IpGeolocationApi3Utilities
  MakeContext = ->(ctxmap, basectx) {
    IpGeolocationApi3Context.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// IpGeolocationApi3 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class IpGeolocationApi3MakeContext
{
    public static function call(array $ctxmap, ?IpGeolocationApi3Context $basectx): IpGeolocationApi3Context
    {
        return new IpGeolocationApi3Context($ctxmap, $basectx);
    }
}
